src/tensorwaves/optimizer/minuit.py

```diff
@@ -69,3 +81,18 @@ def __func(pars: list) -> float:
         results["func_calls"] = function_calls
         results["time"] = end_time - start_time
         return results
+
+
+def dump_parameters(parameters: dict, outputfile: str) -> None:
+    output = {
+        "Parameters": {
+            name: float(value) for name, value in parameters.items()
+        }
+    }
```

Comment:
I would probably just dump the dictionary as it is, without wrapping it in the `"Parameters"` (That's a bit of a preference though). I probably would put parameters in the automated temporary filename. Maybe @redeboer has an opinion about this.

Reply:
I still wrapped the parameters in a subsection, because there's also a `"Time"` section and `"EstimatorValue"`.